reports/pareksha.md

```diff
@@ -0,0 +1,232 @@
+   - details: >
+      Language: Change `TrumpScript` aliases
+ - 5:
+   - repo: coala
+   - link: https://github.com/coala/coala/commit/bcbca4f60363c966f74c52d99a87e543f551e892
+   - details: >
+      Tcl: Create basic language file for tcl
+ - 6:
+   - repo: coala
+   - link: https://github.com/coala/coala/commit/c9455ca65175b4d2ed53e1a9472035f957c8c14f
+   - details: >
+      m4: Create basic language file for m4
+ - 7:
+   - repo: coala
+   - link: https://github.com/coala/coala/commit/c0a78c5c718f93d1bb79926a6e71746b2184a25f
+   - details: >
+      VisualBasic: Create language file for Visual Basic
+ - 8:
+   - repo: coala
+   - link: https://github.com/coala/coala/commit/02aba47f65adea2b0e591ade75be4f65c6319890
+   - details: >
+      PowerShell: Create language file for PowerShell
+ - 9:
+   - repo: coala
+   - link: https://github.com/coala/coala/commit/1aa95e3b08787cce7fb0d5b059a06a513da73735
+   - details: >
+      languages: Allow more than one comment delimiter
+ - 10:
+   - repo: coala-utils
+   - link: https://gitlab.com/coala/coala-utils/commit/26f51a0b82d8738d84d4b5a68305b436dc59f006
+   - details: >
+      position_is_escaped: Let escape char other than `\`
+ - 11:
+   - repo: coala-bears
+   - link: https://github.com/coala/coala-bears/commit/4f1616515730c9caff48d2d593143749a3428f9e
+   - details: >
+      ClangComplexityBear: Fix AttributeError
+ - 12:
+   - repo: coala
+   - link: https://github.com/coala/coala/commit/a04ba9ef871a5e2a589a4645006caf7ab4c82b51
+   - details: >
+      aspects: Rename class `docs` to follow CapWords
+ - 13:
+   - repo: coala
+   - link: https://github.com/coala/coala/commit/b98fdfd2cd05f93b01e921af4f1ee11dc9ef1e29
+   - details: >
+      VisualBasic: Add `max_line_length`
+ - 14:
+   - repo: coala
+   - link: https://github.com/coala/coala/commit/e83dfcc1f0f00cf7aa9abf4f51ef7a6effa804d4
+   - details: >
+      languages: Add missing language imports
+ - 15:
+   - repo: coala
+   - link: https://github.com/coala/coala/commit/656ed289b5a4ab2660e848f409c53256f59dda0a
+   - details: >
+      languages: Implement `get_all_languages()`
+ - 16:
+   - repo: coala
+   - link: https://github.com/coala/coala/commit/0986d56140a7a5e57c97928e9477cc4ac74e73b8
+   - details: >
+      aspects: Default languages=all when using aspects
+ - 17:
+   - repo: coala
+   - link: https://github.com/coala/coala/commit/19e73ee67baf9f36d0dad90f8f7005bdb15235dc
+   - details: >
+      LocalBearTestHelper: Modify for `aspects` testing
+ - 18:
+   - repo: coala-bears
+   - link: https://github.com/coala/coala-bears/commit/6b01667c435df81e8d6a6ac5bf76d061f0b47221
+   - details: >
+      LineLengthBear: Migrate to aspects
+ - 19:
+   - repo: coala-bears
+   - link: https://github.com/coala/coala-bears/commit/c9e8df083c5cae5651b81c0efdc9da3d0f79cd0b
+   - details: >
+      PycodestyleBear: Migrate bear to aspects
+ - 20:
+   - repo: coala-bears
+   - link: https://github.com/coala/coala-bears/pull/2654
```

Comment:
as the three below can all get merged, it is not worth mentioning an unmerged PR here.

describe it in the prose at the bottom, indicating that it is blocked on a bug in an upstream tool